.github/workflows/bbw_build_container_template.yml

```diff
@@ -67,6 +67,9 @@ jobs:
           cd ${{ env.WORKDIR }}
           cat ${{ inputs.dockerfile }} qpress.Dockerfile buildbot-worker.Dockerfile >$GITHUB_WORKSPACE/Dockerfile
           cp -r qpress $GITHUB_WORKSPACE
+          if [ "${{ inputs.nogalera }}" == true ]; then
```

Comment:
Please also set inputs --> nogalera to the header of the template with required `false`